src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -110,20 +110,20 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
         /// <param name="catalog">The categorical transform's catalog.</param>
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
         /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
-        /// <param name="bag">Whether bagging is used for the conversion. </param>
+        /// <param name="useBagging">Whether bagging is used for the conversion. </param>
```

Comment:
>useBagging [](start = 25, length = 10)

I don't like this name at all.
First of all, this parameter make sense only if you apply it to vector of keys.
And it does following:
if you don't set it up, each key in array would produce it's own vector full of zeros with 1 in position of key value, basically you get array of indicators arrays.
If you set it up to true, it will go through array and count how many times in that array you have key of first value, key of second value, etc, and it will produce you array of key counts.

useBagging doesn't reflect that at all. 
[https://www.quora.com/What-is-bagging-in-machine-learning](https://www.quora.com/What-is-bagging-in-machine-learning)
not related at all, right?

Do I have better suggestions?
`OutputCountVector`, maybe, but it doesn't reflect what it got applied only for vector of keys.
`OutptuVectorOfKeysAsCountVector`?

Reply:
You are right! I think we confused with Bag of words, which is a different concept. I like `OutputCountVector`. I have replaced it and changed the documentation.

---
In reply to: [263635509](https://github.com/dotnet/machinelearning/pull/2862#discussion_r263635509) [](ancestors = 263635509)